### PR TITLE
Init all defined buckets on startup

### LINKED ISSUE #494: Initialize all buckets on startup
**Is your feature request related to a problem? Please describe.**
The ability to add custom buckets to the bucket explorer was added but buckets must be manually added before you can use them locally.

**Describe the solution you'd like**
It would be nice to automatically initialize buckets defined in the env file like the original three buckets in `openc3cli.rb`.

**Additional context**
If you define a custom bucket in the env file the bucket explorer will show the bucket but give you an error saying that it doesn't exist because it was never added to minio. You can't upload to custom buckets because of this.

image

### PR DESCRIPTION
**Pull request by wiatrp**
closes #494 

I borrowed this code from what you have in the `files` method in `storage_controller.rb`. Although I combined the two maps into a single one.

### CODE REVIEW

File: openc3/bin/openc3cli [hunk 1]
```diff
@@ -619,9 +619,11 @@ if not ARGV[0].nil? # argument(s) given
 
   when 'initbuckets'
     client = OpenC3::Bucket.getClient()
-    client.create(ENV['OPENC3_CONFIG_BUCKET'])
-    client.create(ENV['OPENC3_LOGS_BUCKET'])
-    client.create(ENV['OPENC3_TOOLS_BUCKET'])
+    ENV.map do |key, value|
+      if key.match(/^OPENC3_(.+)_BUCKET$/) && !value.empty?
+        client.create(value.downcase)
```
Comment (ryanmelt) [inline openc3/bin/openc3cli:624]: Why the downcase?

**Review by ryanmelt — APPROVED** [review]

File: openc3/bin/openc3cli (hunk 1 above)
Comment (ryanmelt) [reply openc3/bin/openc3cli:624]: I think remove that, and this looks good.

**Review by ryanmelt — CHANGES_REQUESTED** [review]
Please remove downcase.

File: openc3/bin/openc3cli (hunk 1 above)
Comment (wiatrp) [reply openc3/bin/openc3cli]: done

**Review by ryanmelt — APPROVED** [review]

**Pull request merged**